erlang/api_cowboy/apps/api_cowboy/src/api_cowboy_handler.erl

```diff
@@ -0,0 +1,145 @@
+-module(api_cowboy_handler).
+-behaviour(cowboy_handler).
+
+-export([init/2]).
+
+init(Req0, State) ->
```

Comment:
As Jose said, you should be using the rest_handler of cowboy, also one important thing for your current implementation, you are not allowing OPTIONS method and that's VERY important in rest calls, but don't worry, rest adapter will take care of that for you.